### PR TITLE
cap pandas version

### PR DESCRIPTION
**Pull request by yinweisu**
*Issue #, if available:*

*Description of changes:*
Capped version of pandas because pandas 1.4.0 because of issue: https://github.com/pandas-dev/pandas/issues/45603

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by Innixma — APPROVED** [review]
LGTM! Lets wait for CI to succeed before merging.

**Comment by szha** [comment]
Job PR-1512-1 is done. 
Docs are uploaded to http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-1512/1/index.html

**Comment by yinweisu** [comment]
Previous CI passed, recent commit just fixed a typo. Can be merged directly

**Pull request merged**
